src/test/scala/org/broadinstitute/dsde/agora/server/business/AgoraBusinessIntegrationSpec.scala

```diff
@@ -22,14 +22,14 @@ class AgoraBusinessIntegrationSpec extends FlatSpec with BeforeAndAfterAll with
     assert(agoraBusiness.findSingle(entity, Seq(entity.entityType.get), mockAutheticatedOwner.get) === entity)
   }
 
-  "Agora" should "be unable to store a task configuration with an invalid docker image (invalid/non-existent repo name)" in {
+  ignore should "be unable to store a task configuration with an invalid docker image (invalid/non-existent repo name)" in {
```

Comment:
This isn't your thing (I think this is fine), but are we running this test twice? Why?